File: Royaleagle73/A1_Q4.clj
(defn get_matched_count [actual_labels predicted_labels]
  (loop [index 0 cnt 0]
    (if (= index (count actual_labels))
      cnt
      (recur (inc index) (if (= (get actual_labels index) (get predicted_labels index)) (inc cnt) cnt)))))

(defn get_accuracy [actual_labels predicted_labels]
  (* 100 (/ (get_matched_count actual_labels predicted_labels) (count actual_labels))))

(defn main [] (
               do (def actual_labels ['Dog','Dog','Cat','Cat','Cat'])
                  (def predicted_labels ['Cat','Dog','Cat','Dog','Cat'])
                  (int (get_accuracy actual_labels predicted_labels))))

(main)

File: tanvigupta27/A1_Q4.py
actual_labels = ['Dog','Dog','Cat','Cat','Cat']
predicted_labels = ['Cat','Dog','Cat','Dog','Cat']
def calc_acc(actual_labels,predicted_labels):
    res = 0
    l = len(actual_labels)
    for i in range(l):
        if actual_labels[i] == predicted_labels[i]:
            res +=1
    
    print(res/l*100)
calc_acc(actual_labels,predicted_labels)   